src/solvers/dg_unstructured_quad/dg_2d.jl

```diff
@@ -221,22 +221,27 @@ function calc_boundary_flux!(cache, t, boundary_condition::BoundaryConditionPeri
 end
 
 
-function calc_boundary_flux!(cache, t, boundary_conditions,
+function calc_boundary_flux!(cache, t, boundary_conditions::NamedTuple,
```

Comment:
```suggestion
function calc_boundary_flux!(cache, t, boundary_conditions,
```
I don't think the type annotation is required here, or is it?

Reply:
This probably stems from my own misunderstanding of type stability. I thought I had to tell the function the explicit type of an arguement to make it type stable. I assume from this comment that I am mistaken?

Reply:
Yeah, annotation is usually just for specifying which function to dispatch. Less restrictive also means you can switch the type of `boundary_conditions` later.

Reply:
Noted. I removed the annotation